app/views/content/is-teaching-right-for-me/career-progression.md

```diff
@@ -0,0 +1,132 @@
+
+## How can you get more responsibilities?
+
+<strong>Fran:</strong> Realistically you’d want to do at least a year or two in the same setting before then thinking about applying for a role with more responsibility. But you have to prove your worth to get a teaching and learning responsibility (TLR) payment. 
+
+The headteacher at my school was looking to create an alternative for key stage 3 (KS3) pupils who weren’t able to access the curriculum properly. As part of a shift in my lead practitioner responsibility for English I was given a permanent TLR to research and implement this. I had to come up with case studies and a proposal. It's about showing you’re prepared to invest in your personal development and know what’s going on out there. You have to keep up with legislation and news and topical priorities. 
+
+<strong>Sian:</strong> It may be the case that your school hasn’t got the budget for a TLR. So if you really want to take on more responsibility you may need to think about moving school.
+
+## How can teachers increase their salary? 
+
+<strong>Fran:</strong> If you have qualified teacher status (QTS) and work in a maintained school, a school funded by the local authority, your salary is reviewed every year. Most teachers on the main pay scale get an annual pay rise but it may depend on your school’s management performance data to prove you’re completing whole school and individual targets. So you could potentially go from a starting salary of £30,000 to over £41,000, or more if you’re in London, in 5 years.
+
+As well as helping you understand the demands of a more senior role, a teaching and learning responsibility (TLR) payment could make a big difference to your pay. If you’re in London it could be around an extra £15,000.
+
+## What steps do you need to consider if you want to be a headteacher?
+
+<strong>Chris:</strong> If you want to go into senior leadership in a secondary school you’ll need to build both a curriculum role and a pastoral role, for example, head of geography and head of year into your early career.
+
+<strong>Elaine:</strong> I've got curriculum and pastoral experience and I also did school timetabling. As a headteacher you need experience of everything to get to the top.
+
+## How long does it take to go from an early career teacher to a headteacher?
+
+<strong>Sian:</strong> I think 8 to 10 years is realistic to become a headteacher for primary schools although you may be able to climb the career ladder a bit quicker.
+
+<strong>Elaine:</strong> If you’re driven you could probably get to headship in between 9 to 12 years. It took me longer but at the time it suited me to stay where I was.
+
+<strong>Chris:</strong> If you want to develop your leadership skills there are [national professional qualification (NPQ) leadership courses](https://www.gov.uk/guidance/national-professional-qualification-npq-courses#leadership-npqs) available to help teachers progress their careers.
```

Comment:
Would it be more appropriate just to link to the page without the anchor, to give the user more context as they might not be familiar with NPQs? Plus some of the specialist NPQs cover some leadership?


```suggestion
<strong>Chris:</strong> If you want to develop your leadership skills there are [national professional qualification (NPQ) courses](https://www.gov.uk/guidance/national-professional-qualification-npq-courses) available to help teachers progress their careers.
```

Reply:
However, I've realised we are linking to the NPQs page twice - do we need to?

If so, the link text needs to be consistent if the destination URL is the same

Reply:
@gemmadallmandfe I'd linked one of them direct to the leadership NPQs so they weren't taking people to exactly the same place. I think I'd kept 2 links in as the plan was to look at the design of this page and potentially have some kind of show and hide component. So it may be that someone wouldn't necessarily see both links. But for the time being I can remove one of the links and just have a general link to NPQs as the page may not get redesigned for a while. And we can look to add one back in then.